### PR TITLE
Redesign add buttons

### PR DESCRIPTION
**Pull request by rubenmoya**
Related to: #13122

## Add new layer
screen shot 2017-12-14 at 13 06 59

## Add new widget
screen shot 2017-12-14 at 13 07 18

## Add new analysis
screen shot 2017-12-14 at 13 07 53

### CODE REVIEW

**Comment by matallo** [comment]
Updated the forms, too:

**analyses**

screen shot 2017-12-18 at 12 09 41

**widgets**

screen shot 2017-12-18 at 12 10 50

**Comment by csobier** [comment]
Please apply the following tooltip edits, for consistency with our terminology. Thank you!

- Add new layer to your dashboard
- Add a new analysis node that is linked to your layer
- Add new widget to your dashboard

**Comment by arianaescobar** [comment]
1. Christa is updating the buttons' tooltips and will add the new copy here 
2. In the forms:

- We are missing the icon when the node is an analysis
- The node code (A1, B0...) should be smaller, as the rest of them. 
- Let's keep the analysis short name + layer name in the larger size in the inputs. Now that we're seeing them _live_ we realized it looks too weird (sorry 😥).
- The last three elements in green are using the old green :( 

screen shot 2017-12-18 at 16 31 46
   
3. There are some other things that I think are not from this PR but are somehow related and I'm not sure where to add them, so feel free to ignore and ask me to open a new issue. 

- In the Widget options, the element below the widget title should have the same structure as everywhere else. So, in the following case, where the widget was created from the A2 analysis, it should also have the icon and the name of the analysis. The layer name should be in grey and not linked. 
 
screen shot 2017-12-18 at 16 17 31

- In these two places (Widget and the cards in the Widgets List), the node + icon + text are not vertically aligned: 

screen shot 2017-12-18 at 16 12 01
screen shot 2017-12-18 at 16 02 29

(And in this last one, the green is the old one).

**Comment by arianaescobar** [comment]
Nodes: 

nodes

**Comment by rubenmoya** [comment]
Thank you @arianaescobar!

**Comment by matallo** [comment]
aaand ready for acceptance again at `ded10` my changes included removing the analysis reference in the edit feature view

**Comment by arianaescobar** [comment]
Just a feeeew things: 

- In Widgets options, the text should be smaller. Look at the table in the comment above.

screen shot 2017-12-20 at 18 36 24

- There's something strange going on here, look: An A0 with the ray icon and an analysis name.
 
screen shot 2017-12-20 at 18 35 56

- It's also happening with other analysis:

node
 
- And a picky thing: Can we prevent this orphan? (As a general rule, we should always try to prevent them)

screen shot 2017-12-20 at 18 41 12

**Comment by arianaescobar** [comment]
Also, for some reason the last layer is not showing the Hide icon:

screen shot 2017-12-20 at 18 48 33

Any idea?

**Comment by rubenmoya** [comment]
The neverending PR 😅  What user did you use so I can download the `.carto`?

**Comment by rubenmoya** [comment]
I've fixed the first two problems, about the other ones:

- With the tooltip, the only thing we can do is increase the minimum size or change the text
- The hidden icon is not showed because the layer has no geometries, if you add an empty layer it won't have the icon either

**Comment by arianaescobar** [comment]
Ooooh got it. 

Regarding the tooltips, can't we do the same as we do on our commercial website to avoid the orphans @matallo?

**Comment by matallo** [comment]
just add a helper which replaces the last space with a `&nbsp;` character in the tooltips @rubenmoya

**Comment by rubenmoya** [comment]
@matallo What a trick!! I didn't know that 😮

File: lib/assets/core/javascripts/cartodb3/helpers/utils.js [hunk 1]
```diff
@@ -291,6 +291,13 @@ module.exports = {
     return str.replace(/(\r\n|\n|\r)/gm, '');
   },
 
+  replaceLastSpaceWithNbsp: function (string) {
```
Comment (rubenmoya) [inline lib/assets/core/javascripts/cartodb3/helpers/utils.js]: Naming is  hard :(

**Comment by arianaescobar** [comment]
Perfect ✨✨✨✨✨✨

**Comment by rubenmoya** [comment]
Vamoooooooos! 🎉

**Comment by rubenmoya** [comment]
retest this please

**Comment by rubenmoya** [comment]
retest this please

**Comment by rubenmoya** [comment]
retest this please

**Comment by rubenmoya** [comment]
retest this please

**Comment by rubenmoya** [comment]
Ready to merge! 🍏

**Pull request merged**